std/traits.d

```diff
@@ -5154,6 +5157,22 @@ unittest
 }
 
 /**
+Detect whether `T` is a scalar type that promotes to an integral type (that is, a built-in
+integral, character, or boolean type).
+ */
+enum bool isFixedPoint(T) = isIntegral!T || isSomeChar!T || isBoolean!T;
```

Comment:
> Is "fixed point" the right term for this?

I intended it as the opposite of `isFloatingPoint`, since `isFixedPoint == (isScalarType && !isFloatingPoint)`. Do you have a suggestion for a better name?